tests/run-test.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# This simply runs a simulator.
+
+INPUT_DATA_PATH="$1"
+RESULT_PATH="$2"
+BINPATH="$3"
+EXE_NAME="$4"
+FILENAME="$5"
+MPI_PROCS="$6"
+shift 7
+TEST_ARGS="$@"
+
+mkdir -p ${RESULT_PATH}
+cd ${RESULT_PATH}
```

Comment:
Is it not too much to cd into that dir? Using it again as output dir leads to norne/norne/ etc.

Reply:
i guess so. this script is not intended for final commit as i (tried to) state above, so i wasn't paying much attention.